pallets/fragments/Cargo.toml

```diff
@@ -103,7 +103,7 @@ version = '4.0.0-dev'
 [dependencies]
 sp-clamor = { version = '0.1.0', path = '../../primitives/clamor', default-features = false }
 hex = { version = "0.4.3", default-features = false }
-serde = { version = "1.0.136", features = ["derive"], optional = true }
+serde = { version = "1.0.136", default-features = false, features = ["derive"]} # , optional = true }
```

Comment:
why we need this when no_std?

Reply:
please add conditional #[derive], don't need to do this

Reply:
We want the serialization of `FragmentInstances` in the **runtime function** `get_instances()`

So we need serde in the runtime/no_std environment